multisig/simulations/vip-004/vip-004-sepolia/simulations.ts

```diff
@@ -0,0 +1,66 @@
+import { mine } from "@nomicfoundation/hardhat-network-helpers";
+import { expect } from "chai";
+import { Contract } from "ethers";
+import { ethers } from "hardhat";
+
+import { NETWORK_ADDRESSES } from "../../../../src/networkAddresses";
+import { initMainnetUser } from "../../../../src/utils";
+import { checkXVSVault } from "../../../../src/vip-framework/checks/checkXVSVault";
+import { forking, pretendExecutingVip } from "../../../../src/vip-framework/index";
+import { vip004 } from "../../../proposals/vip-004/vip-004-sepolia";
+import XVS_ABI from "./abi/xvs.json";
+import XVS_STORE_ABI from "./abi/xvsstore.json";
+import XVS_VAULT_ABI from "./abi/xvsvault.json";
+
+const { sepolia } = NETWORK_ADDRESSES;
+
+forking(4795108, () => {
+  let xvsVault: Contract;
+  let xvsStore: Contract;
+  let xvs: Contract;
+
+  before(async () => {
+    xvsVault = await ethers.getContractAt(XVS_VAULT_ABI, sepolia.XVS_VAULT_PROXY);
+    xvsStore = await ethers.getContractAt(XVS_STORE_ABI, sepolia.XVS_STORE);
+    xvs = await ethers.getContractAt(XVS_ABI, sepolia.XVS);
+    await pretendExecutingVip(vip004());
+
+    // Set reward per block for post checks
+    const admin = await initMainnetUser(sepolia.NORMAL_TIMELOCK, ethers.utils.parseEther("1"));
+    await xvsVault.connect(admin).setRewardAmountPerBlock(sepolia.XVS, "61805555555555555");
```

Comment:
Are these two statements needed, given they are already in the `before` block of the `Generic checks` test, where the generic XVSVault check is performed?